xsd/benchmark/benchmark-8.xsd

```diff
@@ -1,877 +1,2633 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tns="https://www.optaplanner.org/xsd/benchmark" elementFormDefault="qualified" targetNamespace="https://www.optaplanner.org/xsd/benchmark" version="1.0">
+      
+  
+    
+  
```

Comment:
@rsynek All these empty lines in the new xsd don't seem quite right.

Reply:
Sure they don't, but we agreed to leave it as the XML libraries generate it. As an alternative, we would have to remove the empty lines in the CI.